x-pack/plugins/cases/common/api/cases/index.ts

```diff
@@ -12,3 +12,4 @@ export * from './status';
 export * from './user_actions';
 export * from './constants';
 export * from './alerts';
+export * from './suggest_user_profiles';
```

Comment:
nit: As probably more types of user profiles will arise maybe it is better to name the file `user_profiles`